### ISSUE TITLE
In Permanent Private Mode implement containers without needing to set it up every time

### ISSUE THREAD

**Issue by igor2x** [mozilla/testpilot-containers#489]
May I suggest the following. Implement Containers (or consider it) in Firefox in the following way (specially in permanent Private Mode):
1. New "+" is clicked to open new tab ALWAYS open new container (every new tab new container). Don't limit to 4 containers, just every new tab new container. Don't ask user about containers, just open new tab in new container.
2. Executing new Bookmark from browser ALWAYS open new container. The same don't limit to 4 containers. Don't ask users about containers, just open new containers.
3. If web page is opened like right mouse click and selecting "Open Link in New Tab" or clicking middle mouse wheel and open new page in the SAME container. Opening this way it is obvious user wants to continue working on existing web page.

Maybe add extension options:
a) Add new/same container on new tab.
b) Add new/same container if launched from browser.
c) Add new/same container if launched from existing web page.
So basically three options to choose from and making a) and b) default "new" and c) same.

I don't know for others, but in PrivateMode it is obvious user enters userid and password to every site to preserve privacy - automatic login is not desired.

I understand it really depends what end-user wants. For example login into Gmail and in new tab Youtube to be both connected with one single login. Probably yes, but in PrivateMode it is more obvious users care about privacy and do not want to share the passwords between tabs.

Mark this bug report as enhancement or something, not really a bug report.

**Comment by jonathanKingston** [comment]
Seems like a duplicate of #490 no? please reopen if it isn't.
